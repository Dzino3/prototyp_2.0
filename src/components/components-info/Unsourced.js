import React from 'react'

function Unsourced(unsourcedInfo) {
  return (
    <div>
       <div className='orderInfo'>
            <div className='infoUnsourcedLabel'>
                <h3 className='orderH3'>Unsourced</h3>
                <label className='reqInfo'>RDD:</label>
                <label className='reqInfo'>Req. ship date:</label>
                <label className='reqInfo'>Pres date:</label>
                <label className='reqInfo'>Base Code:</label>
                <label className='reqInfo'>Qty.:</label>
                <label className='reqInfo'>Order entry date:</label>
                <label className='reqInfo'>Order Source date:</label>
                <label className='reqInfo'>Order staus:</label>
            </div>
            <div className='infoUnsourced'>
                <h3 className='orderH3i'>Info</h3>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.RDD} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.REQ_SHIP_DATE} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.PRES_DATE} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.BASE_CODE} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.QTY} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.ORDER_ENTRY_DATE} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.ORDER_SOURCE_DATE} disabled></input>
                <input type='text' name='info' id='ACCinfo' className='reqInfoOut' placeholder={unsourcedInfo.unsourcedInfo.ORDER_STATUS} disabled></input>
            </div>
        </div>
    </div>
  )
}

export default Unsourced
