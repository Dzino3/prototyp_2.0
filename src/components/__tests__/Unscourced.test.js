import React from "react";
import renderer from 'react-test-renderer';
import Unsourced from '../components-info/Unsourced';

describe(`Rendering Unsourced`, () => {

    test('should render Unsourced with UnsourcedInfo unsourced', function () {
        let unsourcedInfoX = {
            Unsourced
        }
        let tree = renderer.create(<Unsourced unsourcedInfo={unsourcedInfoX}></Unsourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });

    test('should render OrderListItemSourced with no unsourcedInfo', function () {
        let unsourcedInfoX = {}
    
        let tree = renderer.create(<Unsourced unsourcedInfo={unsourcedInfoX}></Unsourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });

})