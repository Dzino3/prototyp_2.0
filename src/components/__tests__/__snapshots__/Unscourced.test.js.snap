// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Unsourced should render OrderListItemSourced with no unsourcedInfo 1`] = `
<div>
  <div
    className="orderInfo"
  >
    <div
      className="infoUnsourcedLabel"
    >
      <h3
        className="orderH3"
      >
        Unsourced
      </h3>
      <label
        className="reqInfo"
      >
        RDD:
      </label>
      <label
        className="reqInfo"
      >
        Req. ship date:
      </label>
      <label
        className="reqInfo"
      >
        Pres date:
      </label>
      <label
        className="reqInfo"
      >
        Base Code:
      </label>
      <label
        className="reqInfo"
      >
        Qty.:
      </label>
      <label
        className="reqInfo"
      >
        Order entry date:
      </label>
      <label
        className="reqInfo"
      >
        Order Source date:
      </label>
      <label
        className="reqInfo"
      >
        Order staus:
      </label>
    </div>
    <div
      className="infoUnsourced"
    >
      <h3
        className="orderH3i"
      >
        Info
      </h3>
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
    </div>
  </div>
</div>
`;

exports[`Rendering Unsourced should render Unsourced with UnsourcedInfo unsourced 1`] = `
<div>
  <div
    className="orderInfo"
  >
    <div
      className="infoUnsourcedLabel"
    >
      <h3
        className="orderH3"
      >
        Unsourced
      </h3>
      <label
        className="reqInfo"
      >
        RDD:
      </label>
      <label
        className="reqInfo"
      >
        Req. ship date:
      </label>
      <label
        className="reqInfo"
      >
        Pres date:
      </label>
      <label
        className="reqInfo"
      >
        Base Code:
      </label>
      <label
        className="reqInfo"
      >
        Qty.:
      </label>
      <label
        className="reqInfo"
      >
        Order entry date:
      </label>
      <label
        className="reqInfo"
      >
        Order Source date:
      </label>
      <label
        className="reqInfo"
      >
        Order staus:
      </label>
    </div>
    <div
      className="infoUnsourced"
    >
      <h3
        className="orderH3i"
      >
        Info
      </h3>
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
    </div>
  </div>
</div>
`;
